container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -114,7 +114,7 @@ instance_groups:
 
       properties.db_password: ((MYSQL_ADMIN_PASSWORD))
       properties.endpoint_password: ((MYSQL_GALERA_HEALTHCHECK_ENDPOINT_PASSWORD))
-      properties.engine_config.binlog.enabled: false
+      properties.engine_config.binlog.enabled: true
```

Comment:
This options is [default enabled](https://github.com/cloudfoundry-incubator/pxc-release/blob/v0.18.0/jobs/pxc-mysql/spec#L177); just removing the override should be enough?

(For reference, it looks like galera is [off by default](https://github.com/cloudfoundry-incubator/pxc-release/blob/v0.18.0/jobs/pxc-mysql/spec#L252) so keeping that one around is fine.)

Reply:
Can do that. For the original test I wanted to be explicit despite it being default on.

Reply:
Done with commit a6a5716621d9e88e0fd511d0da0fd79ae0542502.
Also done for the SCF PR (updated to commit, and done the same change for itself).